sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,31 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
```

Comment:
All of the configuration settings below don't actually describe what they are, how they should be used, what the parameter type is and what units are represented. For example, this really needs more detail:
```
- `Retry Fixed Interval`: can be changed by passing the `retry_fixed_interval` option. This option has no default value,
and is only present if it is passed in by the user.
```
Maybe something like:
```
- `Retry Fixed Interval`: Represents a fixed wait period between retry attempts in seconds. Can be changed by passing the `retry_fixed_interval` option (`float`). The default behaviour is to use an exponential retry policy. Must be a value >=1.
```